### PR TITLE
Remove native_layer_norm since it's upstreamed.

### PR DESCRIPTION
**Pull request by ailzhang**

### CODE REVIEW

File: torch_xla/csrc/aten_xla_type.cpp [hunk 1]
```diff
@@ -2235,13 +2235,6 @@ AtenXlaType::native_group_norm_backward(
       grad_out, input, mean, rstd, weight, N, C, HxW, group, output_mask);
 }
 
-std::tuple<at::Tensor, at::Tensor, at::Tensor> AtenXlaType::native_layer_norm(
-    const at::Tensor& input, const c10::optional<at::Tensor>& weight,
-    const c10::optional<at::Tensor>& bias, int64_t M, int64_t N, double eps) {
-  XLA_FN_COUNTER("xla::");
-  return aten_tensor_ops::native_layer_norm(input, weight, bias, M, N, eps);
```
Comment (JackCaoG) [inline torch_xla/csrc/aten_xla_type.cpp:2242]: Should we remove `native_layer_norm` implementation as well?

Comment (ailzhang) [reply torch_xla/csrc/aten_xla_type.cpp:2242]: Ah nice catch! didn't realize that I didn't add those changes in lol.

File: torch_xla/csrc/aten_xla_type.cpp [hunk 2]
```diff
@@ -2235,13 +2235,6 @@ AtenXlaType::native_group_norm_backward(
...
-}
-
 std::tuple<at::Tensor, at::Tensor, at::Tensor>
 AtenXlaType::native_layer_norm_backward(
```
Comment (JackCaoG) [inline torch_xla/csrc/aten_xla_type.cpp:2239]: Do we still need this backward? I saw in the test you updated the counter check to `native_batch_norm_backward `

Comment (ailzhang) [reply torch_xla/csrc/aten_xla_type.cpp:2239]: No we don't need them I think. But I'd like to remove them in a followup PR just in case ;)

**Review by JackCaoG — APPROVED** [review]
Thanks!

**Pull request merged**